app/controllers/admin/dashboard_controller.rb

```diff
@@ -17,4 +18,10 @@ def valid_type?(type)
       :devices_detection]
     )
   end
+
+  def getVisits()
+    query = Piwik::VisitsSummary.getVisits(:idSite => 6, :period => :month, :date => :last12)
+    return query.result.to_json
```

Comment:
Ruby automatically returns what's at the end of methods so we don't necessarily have to have a `return` statement here

Reply:
Fixed, deleted this function